test/aes_test.exs

```diff
@@ -1,42 +1,45 @@
 defmodule Fields.AESTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: true
   alias Fields.AES
 
   doctest Fields.AES
 
-  test ".encrypt can encrypt a value" do
-    assert AES.encrypt("hello") != "hello"
-  end
-
-  test ".encrypt can encrypt a number" do
-    assert is_binary(AES.encrypt(123))
-  end
+  describe "encrypt/1" do
+    test "can encrypt a value" do
+      assert AES.encrypt("hello") != "hello"
+    end
 
-  test ".encrypt includes the random IV in the value" do
-    <<iv::binary-16, ciphertext::binary>> = AES.encrypt("hello")
+    test "can encrypt a number" do
+      assert is_binary(AES.encrypt(123))
+    end
 
-    assert String.length(iv) != 0
-    assert String.length(ciphertext) != 0
-    assert is_binary(ciphertext)
-  end
+    test "includes the random IV in the value" do
+      <<iv::binary-16, ciphertext::binary>> = AES.encrypt("hello")
 
-  test ".encrypt does not produce the same ciphertext twice" do
-    assert AES.encrypt("hello") != AES.encrypt("hello")
-  end
+      assert String.length(iv) != 0
+      assert String.length(ciphertext) != 0
+      assert is_binary(ciphertext)
+    end
 
-  test "can decrypt a value" do
-    plaintext = "hello" |> AES.encrypt() |> AES.decrypt()
-    assert plaintext == "hello"
+    test "does not produce the same ciphertext twice" do
+      assert AES.encrypt("hello") != AES.encrypt("hello")
+    end
   end
 
-  test "can decrypt/2 a value" do
-    ciphertext = AES.encrypt("hello")
-    plaintext = AES.decrypt(ciphertext)
-    assert plaintext == "hello"
-  end
 
-  test "decrypt/1 ciphertext that was encrypted with default key" do
-    plaintext = "hello" |> AES.encrypt() |> AES.decrypt()
-    assert plaintext == "hello"
+  describe "decrypt/1" do
+    test "should be able to decrypt an encrypted value" do
+      encrypted = AES.encrypt("hello")
+      assert "hello" == AES.decrypt(encrypted)
+    end
+
+    test "should always be able to decrypt the values in the same way (no breaking changes)" do
```

Comment:
I added this test before the API update, to make sure the new API is decrypting values generated with the `:crypto.block_encrypt`, preventing breaking changes.